src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -396,6 +397,8 @@ def get_docker_command(is_diagnostics_context=False):
         docker_command = os.getenv('DOCKER_COMMAND')
     else:
         docker_command = 'docker'
+        if not shutil.which('docker') and shutil.which('podman'):
```

Comment:
This method is also used by [_get_docker_status_and_version](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acr/check_health.py#L77C5-L77C35) which outputs hardcoded "Docker" messages (e.g., "Docker version", "Docker daemon status"). When Podman is used instead, these messages will be inaccurate. Please update check_healthy.py as well to dynamically reflect which tool is being checked, or use tool-agnostic terminology like "Container runtime".